### PR TITLE
Transition to use make-env

### PR DESCRIPTION
**Pull request by acomodi**
This PR should fix https://github.com/SymbiFlow/python-symbiflow-v2x/issues/105.

It allows to create an environment to use v2x by running `make env`

### CODE REVIEW

File: .github/workflows/licensing.yml [hunk 1]
```diff
@@ -1,24 +0,0 @@
-# Copyright (C) 2017-2021  The SymbiFlow Authors.
```
Comment (mithro) [inline .github/workflows/licensing.yml:1]: Why is the licensing check being removed?

Comment (mithro) [reply .github/workflows/licensing.yml:1]: Looks like it is because it is now done by the `ci.yml`?

File: docs/environment.yml [hunk 2]
```diff
@@ -1,3 +1,11 @@
+# Copyright (C) 2021  The Symbiflow Authors.
```
Comment (mithro) [inline docs/environment.yml:1]: I think you'll need to fix line 28 to fix ReadTheDocs?

**Review by mithro — APPROVED** [review]
Once you get ReadTheDocs working, go ahead and merge!

File: .github/workflows/licensing.yml (hunk 1 above)
Comment (acomodi) [reply .github/workflows/licensing.yml:1]: Indeed, the licenses and python checks were executed twice

**Pull request merged**